src/analysis/special_functions/trigonometric/angle.lean

```diff
@@ -271,6 +272,131 @@ cos_antiperiodic θ
 @[simp] lemma cos_sub_pi (θ : angle) : cos (θ - π) = -cos θ :=
 cos_antiperiodic.sub_eq θ
 
+@[simp] lemma coe_to_Ico_mod (θ ψ : ℝ) : ↑(to_Ico_mod ψ two_pi_pos θ) = (θ : angle) :=
+begin
+  rw angle_eq_iff_two_pi_dvd_sub,
+  refine ⟨to_Ico_div ψ two_pi_pos θ, _⟩,
+  rw [to_Ico_mod_sub_self, zsmul_eq_mul, mul_comm]
+end
+
+@[simp] lemma coe_to_Ioc_mod (θ ψ : ℝ) : ↑(to_Ioc_mod ψ two_pi_pos θ) = (θ : angle) :=
+begin
+  rw angle_eq_iff_two_pi_dvd_sub,
+  refine ⟨to_Ioc_div ψ two_pi_pos θ, _⟩,
+  rw [to_Ioc_mod_sub_self, zsmul_eq_mul, mul_comm]
+end
+
+/-- Convert a `real.angle` to a real number in the interval `Ioc (-π) π`. -/
+def to_real (θ : angle) : ℝ :=
+(to_Ioc_mod_periodic (-π) two_pi_pos).lift θ
+
+lemma to_real_coe (θ : ℝ) : (θ : angle).to_real = to_Ioc_mod (-π) two_pi_pos θ :=
+rfl
+
+lemma to_real_coe_eq_self_iff {θ : ℝ} : (θ : angle).to_real = θ ↔ -π < θ ∧ θ ≤ π :=
+begin
+  rw [to_real_coe, to_Ioc_mod_eq_self two_pi_pos],
+  ring_nf
+end
+
+lemma to_real_coe_eq_self_iff_mem_Ioc {θ : ℝ} : (θ : angle).to_real = θ ↔ θ ∈ set.Ioc (-π) π :=
+by rw [to_real_coe_eq_self_iff, ←set.mem_Ioc]
+
+lemma to_real_injective : function.injective to_real :=
+begin
+  intros θ ψ h,
+  induction θ using real.angle.induction_on,
+  induction ψ using real.angle.induction_on,
+  simpa [to_real_coe, to_Ioc_mod_eq_to_Ioc_mod, zsmul_eq_mul, mul_comm _ (2 * π),
+         ←angle_eq_iff_two_pi_dvd_sub, eq_comm] using h,
+end
+
+@[simp] lemma coe_to_real (θ : angle): (θ.to_real : angle) = θ :=
+begin
+  induction θ using real.angle.induction_on,
+  exact coe_to_Ioc_mod _ _
+end
+
+lemma neg_pi_lt_to_real (θ : angle) : -π < θ.to_real :=
+begin
+  induction θ using real.angle.induction_on,
+  exact left_lt_to_Ioc_mod _ two_pi_pos _
+end
+
+lemma to_real_le_pi (θ : angle) : θ.to_real ≤ π :=
+begin
+  induction θ using real.angle.induction_on,
+  convert to_Ioc_mod_le_right _ two_pi_pos _,
+  ring
+end
+
+lemma abs_to_real_le_pi (θ : angle) : |θ.to_real| ≤ π :=
+abs_le.2 ⟨(neg_pi_lt_to_real _).le, to_real_le_pi _⟩
+
+lemma to_real_mem_Ioc (θ : angle) : θ.to_real ∈ set.Ioc (-π) π :=
```

Comment:
Isn't this immediate from `neg_pi_lt_to_real` and `to_real_le_pi`?

Reply:
I've changed the proof to use those lemmas.